pkg/vault/contracts/authorizer/TimelockAuthorizer.sol

```diff
@@ -360,8 +360,7 @@ contract TimelockAuthorizer is IAuthorizer, IAuthentication {
         address[] memory executors
     ) external returns (uint256 scheduledExecutionId) {
         require(newDelay <= MAX_DELAY, "DELAY_TOO_LARGE");
-        bool isAllowed = _hasPermissionOrWhatever(SCHEDULE_DELAY_ACTION_ID, msg.sender, address(this), actionId);
```

Comment:
With this change we no longer meaningfully use `SCHEDULE_DELAY_ACTION_ID` (only for emitting events) - does it make sense to keep it?